utils/build/docker/php/common/user_login_failure_event.php

```diff
@@ -1,5 +1,6 @@
 <?php
-\datadog\appsec\track_user_login_failure_event('system_tests_user', true,
+// pass user ID as integer to test that it ends up in meta not metrics
+\datadog\appsec\track_user_login_failure_event(123456, true,
 [
     'metadata0' => 'value0',
```

Comment:
If this is the data that are being sent in meta you'd need to add 'usr.id' => 123456. What I want is for this login to send inside meta this "appsec.events.users.login.failure.usr.id": 123456